src/command_line_parser.cc

```diff
@@ -1688,13 +1716,14 @@ CLParser::ParseCommandLine(int argc, char** argv)
       }
     }
     catch (const std::invalid_argument& ia) {
-      if (opt >= 'A') {  // short options
+      if (opt < long_option_idx_base) {  // short options
```

Comment:
Reworked logic here because as mentioned [here](https://github.com/triton-inference-server/perf_analyzer/pull/74/files#r1744758068), we start all long options at 256 now, and all options below that are short options.